README: Adding references to the website (#536)
clusterlink-net/clusterlink: README.md

# ClusterLink Project
# ClusterLink

> ## Disclaimers and Warnings

Reviewer: I think we should keep the caveat for quite some time still. It can be reworded, but not removed

> For more details, see our website: [clusterlink.net](https://clusterlink.net/).

Reviewer: ```suggestion
For more details, visit our [website](https://clusterlink.net/).
```

1. ```Control Plane``` is responsible for maintaining the internal state of the gateway,
 for all the communications with the remote peer gateways by means of the ClusterLink CP

>  Protocol, and for commanding the local DP to forward user traffic according

Reviewer: ```suggestion
 Protocol, and for configuring the local data plane to forward user traffic according
```

 presenting itself as a K8s service inside the cluster and as a regular HTTP endpoint
 from outside the cluster, requiring only a single open port (HTTP/443) and leveraging
 HTTP endpoints for connection multiplexing.

![alt text](./docs/figures/clusterlink.png)

> ClusterLink leverages Kubernetes CRDs and the kubectl API to configure cross-cluster communication.

Reviewer: ```suggestion
ClusterLink leverages the Kubernetes API using CRDs to configure cross-cluster communication.
```
There is no kubectl  API. It is the CLI to the k8s API

> ClusterLink utilizes the following key concepts:

Reviewer: ```suggestion
ClusterLink management is based on the following key concepts:
```

- Policy. Represent communication rules that must be enforced for all cross-cluster
 communications at each ClusterLink gateway.

> For further information, please refer to the [concepts section](https://clusterlink.net/docs/main/concepts/)) on the ClusterLink website.

Reviewer: nit: you should likely not refer to 'version` in the URL. If anything, I would recommend the latest released version,,,
But it may be best if we add redirects from /docs/* to /docs/latest/* in netlify.
I still need to figure it out and then configure it...

Reviewer: Also, I started using named links and putting all URL at the bottom of the markdown doc. This makes later updates easier as they are not mixed with text.
The format is one of:
- `[link text][link name]`; or
- `[link text][]` (empty brackets imply `link name` == `link text`)

Either should by followed by `[link name]: URL` at the bottom. Link names may contain spaces. You can have multiple links (possibly with different text) referencing the same URL, defined once.

Author: For now, I use main until we have the latest redirect

> ## Getting Started

Reviewer: ```suggestion
## Getting started
```
Keep capitalization consistent

ClusterLink can be set up and run on different environments: local environment (Kind),
 Bare-metal environment, or cloud environment. For more details, refer to the [Getting Started Guide]([docs/installation.md](https://clusterlink.net/docs/main/getting-started/users/)).

Additionally, here are some other documents you may find helpful:

- [Cluster Tutorials](https://clusterlink.net/docs/main/tutorials/) - These tutorials describe how to establish multi-cluster connectivity for applications using two or more clusters.

> - [ClusterLink Developer's Guide](https://clusterlink.net/docs/main/getting-started/developers/) -This guide explains how to configure a development environment and contribute code to ClusterLink.

Reviewer: ditto, regarding the use of "versions" in links and using named links
